src/controllers/module/event/eventController.ts

```diff
@@ -128,6 +127,55 @@ export class EventController {
 		}
 	}
 
+	public getLocation() {
+		return async (req: Request, res: Response) => {
+			res.render('page/course/module/events/event-location', {event: req.session!.event})
+		}
+	}
+
+	public setLocation() {
+		return async (req: Request, res: Response) => {
+			const data = {venue: {
+					location: req.body.location,
+					address: req.body.address,
+					capacity: parseInt(req.body.capacity, 10),
+					minCapacity: parseInt(req.body.minCapacity, 10),
+				}}
+
+ 			const errors = await this.eventValidator.check(data, ['event.location'])
+			const event = await this.eventFactory.create(data)
+
+			if (errors.size) {
+				req.session!.sessionFlash = {errors: errors}
+				req.session!.event = req.session!.event ? function() {
+					const mergedEvent = req.session!.event
+					mergedEvent.venue = event.venue
+					return mergedEvent
+				}() : event
```

Comment:
The formatting done by prettier here is a bit messy, could we maybe use a `// prettier-ignore` on line 150?

Reply:
what's `prettier`? it was me :-D unless you refer to some intermediate results

Reply:
Prettier is our auto-formatting tool that we use in intelliJ, can you check you have this set up?